bentoml/_internal/frameworks/transformers.py

```diff
@@ -63,10 +62,8 @@
 
 MODULE_NAME = "bentoml.transformers"
 
-try:
-    from huggingface_hub import HfFolder
-except ImportError:
-    HfFolder = None
+# `save` should only be used when training or working with a customized pretrained transformers model
```

Comment:
remove? or move to the `save` method doc string

Reply:
I will move it to docstring